tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+- **Redshift UserId/Password combination** - You will need to provide this information when the Redshift service was provisioned.
+- **Redshift URL** - You will need to obtain this from your Redshift Console. A sample Redshift URL is `swredshift.czac2vcs84ci.us-east-1.redshift.amazonaws.com:5439`
+
+Your Redshift console will provide the JDBC URL to use. It follows the pattern
+
+`jdbc:redshift://$redshifturl/$redshiftDBName?user=$redshiftUserId&password=$redshiftPassword`
+
+A sample JDBC URL is
+
+`jdbc:redshift://swredshift.czac2vcs84ci.us-east-1.redshift.amazonaws.com:5439/sparkredshift?user=spark&password=mysecretpass`
+
+spark-redshift reads and writes data to S3 when transferring data to/from Redshift, so you'll need to specify a path in S3 where the library should write these temporary files. spark-redshift cannot automatically clean up the temporary files it creates in S3. As a result, we recommend that you use a dedicated temporary S3 bucket with an [object lifecycle configuration ](http://docs.aws.amazon.com/AmazonS3/latest/dev/object-lifecycle-mgmt.html) to ensure that temporary files are automatically deleted after a specified expiration period. For this example we create a S3 bucket `redshift-spark`. We tell spark-redshift that we will use the following temporary location in S3 to store temporary files generated by spark-redshift `s3n://redshift-spark/temp/`
+
+Next enable the communication with AWS by configuring the following properties in the `SparkContext` instance-
+
+- fs.s3n.awsAccessKeyId 
+- fs.s3n.awsSecretAccessKey
+
+Lastly we create the SQLContext to use the Data Source API to communicate with Redshift
+
+`val sqlContext = new org.apache.spark.sql.SQLContext(sc)`
+
+### Load Function - Reading from a Redshift table###
+
+Let us fetch data from the Redshift table `event`. Add the following lines of code listed in Listing 2 to the program in Listing 1
+
+```scala
+import sqlContext.implicits._
+val eventsDF = sqlContext.read
+               .format("com.databricks.spark.redshift")
+               .option("url",jdbcURL ) 
+               .option("tempdir", tempS3Dir) 
+               .option("dbtable", "event")
+               .load()
+eventsDF.show()   
+```
+Listing 2 : Reading an entire Redshift table 
+
+`.format("com.databricks.spark.redshift")` line tells the DataSource API that we are using the `spark-redshift` package. It uses this information to identify the class `DefaultSource` in the package specified by the `format` invocation. This class contains the entry points for the functionality provided by the Data Sources API implementation.
+
+Next we provide the parameters necessary to read the `event` table. We provide the JDBC URL, the temporary S3 folder which will be used to copy the table data to and the table name we want to read. A comprehensive list of parameters is listed on the spark-redshift documentation [page](https://github.com/databricks/spark-redshift#parameters).
```

Comment:
Minor nit, but I'd put all of `"spark-redshift documentation page" in the clickable portion of the link. Friendlier to screen-readers and more obviously clickable.